### PR TITLE
Space UPS

### PR DESCRIPTION
**Pull request by alexkar598**
:cl: alexkar598
add: Added Item delivering,go to station A,get thing,get to B,deliver thing. 
/:cl:

its broken and needs fixing,you cant teleport stiff you pull,review this pr anyways.

### CODE REVIEW

**Comment by Kmc2000** [comment]
nice mapmerge

File: DS13/code/modules/overmap/events.dm [hunk 1]
```diff
@@ -351,4 +351,117 @@ GLOBAL_LIST_INIT(overmap_event_spawns, list())
...
+
+
+
+//----------------------
+//ITEM DELIVERY AKA SPACE UPS
+//----------------------
+
+//LANDMARK
+
+GLOBAL_DATUM(sealedcratespawn,/obj/effect/landmark/sealedcrate)
```
Comment (Kmc2000) [inline DS13/code/modules/overmap/events.dm:365]: This isnt a datum?

File: DS13/code/modules/overmap/events.dm [hunk 2]
```diff
@@ -351,4 +351,117 @@ GLOBAL_LIST_INIT(overmap_event_spawns, list())
...
+	density = TRUE
+
+/obj/structure/sealedcrate/attack_hand(mob/user)
+	. = ..()
+	if(.)
+		return
+	to_chat(user,"<span class='warning'>This crate is locked and sealed,you can't open it.</span>") //HEY LEAVE THIS ALONE OR YOUR SCRUBBING PLASMA CONDUITS
+
+/obj/structure/sealedcrate/Destroy()
+	. = ..()
```
Comment (Kmc2000) [inline DS13/code/modules/overmap/events.dm:395]: This should go after, as it deletes the crate

File: DS13/code/modules/overmap/events.dm [hunk 3]
```diff
@@ -351,4 +351,117 @@ GLOBAL_LIST_INIT(overmap_event_spawns, list())
...
+	icon = 'DS13/icons/obj/cratereceiver.dmi'
+	icon_state = "delivery_point"
+	density = FALSE
+	opacity = FALSE
+
+/obj/structure/crate_receiver/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	if(istype(AM,/obj/structure/sealedcrate))
+		var/obj/structure/sealedcrate/SC = AM
+		SC.linked_event.succeed()
```
Comment (Kmc2000) [inline DS13/code/modules/overmap/events.dm:413]: if(SC.linked_event)

**Review by Kmc2000 — CHANGES_REQUESTED** [review]

File: DS13/code/modules/overmap/events.dm (hunk 1 above)
Comment (alexkar598) [reply DS13/code/modules/overmap/events.dm:365]: It is,datum is the parent type for all objects

File: DS13/code/modules/overmap/events.dm (hunk 3 above)
Comment (alexkar598) [reply DS13/code/modules/overmap/events.dm:413]: It shouldn't ever exist without a linked_event.

**Comment by Kmc2000** [comment]
https://github.com/DDMers/Deep-Space-13/pull/82

**Pull request closed without merging**